### PR TITLE
doc: adding CELO-OLAS pair info

### PR DESCRIPTION
**Pull request by kupermind**
- Adding CELO-OLAS pair info.

### CODE REVIEW

File: docs/lp_token_bridging.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ in order to participate in bonding.
...
\ No newline at end of file
+| Base        | [Balancer](https://app.balancer.fi/#/base/pool/0x5332584890d6e415a6dc910254d6430b8aab7e69000200000000000000000103)         | OLAS-USDC   | [0x5332584890d6e415a6dc910254d6430b8aab7e69](https://basescan.org/address/0x5332584890d6e415a6dc910254d6430b8aab7e69)             | [0x9946d6FD1210D85EC613Ca956F142D911C97a074](https://etherscan.io/token/0x9946d6FD1210D85EC613Ca956F142D911C97a074) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
+| Celo        | [Ubeswap](https://app.ubeswap.org/#/swap)          | CELO-OLAS       | [0x2976Fa805141b467BCBc6334a69AffF4D914d96A](https://celoscan.io/address/0x2976Fa805141b467BCBc6334a69AffF4D914d96A)             | [0xC085F31E4ca659fF8A17042dDB26f1dcA2fBdAB4](https://etherscan.io/token/0xc085f31e4ca659ff8a17042ddb26f1dca2fbdab4) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
```
Comment (kupermind) [inline docs/lp_token_bridging.md:15]: Celo info added

**Review by 77ph — DISMISSED** [review]

File: docs/lp_token_bridging.md (hunk 1 above)
Comment (DavidMinarsch) [reply docs/lp_token_bridging.md:15]: can the swap url not point directly to our pair? see how base points to the pool

Comment (kupermind) [reply docs/lp_token_bridging.md:15]: The Base is still pointing to the pool UI. I've spent a lot of time trying to do it at least how the Uniswap is done, but it doesn't work. If it's fine to have just a pair address, we can do this.

Comment (77ph) [reply docs/lp_token_bridging.md:15]: https://info-v2.ubeswap.org/pair/0x2976Fa805141b467BCBc6334a69AffF4D914d96A maybe?

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**